app/templates/_react.html

```diff
@@ -0,0 +1,11 @@
+{% extends "_base_page.html" %}
+
+{% block body_end %}
+  <script type="text/javascript" src="{{ component.get_vendor_bundle() }}"></script>
+  <script type="text/javascript" src="{{ component.get_bundle() }}"></script>
+{% endblock %}
+
+{% block main_content %}
```

Comment:
And probably also https://github.com/AusDTO/dto-digitalmarketplace-buyer-frontend/pull/381/files#diff-6a07844c11d1c8086e86f1c8e1a5f2cdR4